### ISSUE TITLE
add user accounts and authentication

### ISSUE THREAD

**Issue by peterVG** [artefactual-labs/AIPscan#17]

**Comment by scollazo** [comment]
This could be handled at first by using nginx and http authentication

**Comment by peterVG** [comment]
Yes, Flask provides an easy way to incorporate http authentication using an ENV variable for un/pw. We can implement that as a first simple step before incorporating user accounts.
